modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -380,18 +424,53 @@ export default class ParticipantConnectionStatus {
             isConnActiveByJvb = true;
         }
 
-        const isConnectionActive
-            = isConnActiveByJvb
-                && (isVideoMuted || (isInLastN && !isVideoTrackFrozen));
+        let newState = ParticipantConnectionStatus.INACTIVE;
+
+        if (isConnActiveByJvb) {
+            if (isInLastN) {
```

Comment:
Can you explain what you mean. When a participant is not in lastN it is inactive, and no video should be shown, the UI will show avatar and appropriate message. Inactive is not a connectivity issue.